histomicstk/cli/SuperpixelSegmentation/SuperpixelSegmentation.py

```diff
@@ -0,0 +1,225 @@
+import json
+import logging
+import math
+import os
+
+import large_image
+import numpy
+import pyvips
+import scipy
+import skimage
+from histomicstk.cli import utils
+from histomicstk.cli.utils import CLIArgumentParser
+
+logging.basicConfig()
+
+
+def createSuperPixels(opts):  # noqa
+    averageSize = opts.superpixelSize ** 2
+    overlap = opts.superpixelSize * 4 * 2 if opts.overlap else 0
+    tileSize = opts.tileSize + overlap
+    boundaries = opts.boundaries
+
+    print('>> Reading input image')
+    print(opts.inputImageFile)
+
+    ts = large_image.getTileSource(opts.inputImageFile)
```

Comment:
`getTileSource` and `open` are equivalent, but I think `open` is clearer.

Reply:
Switched to `open` in 1fe8ac5